core/modules/system/system.module

```diff
@@ -2948,10 +2948,12 @@ function _system_rebuild_module_data() {
           }
         }
         if ($git_version) {
-          $git_version = preg_replace('/^' . preg_quote(BACKDROP_CORE_COMPATIBILITY, '/') . '-/', '', $git_version);
           $modules[$key]->info['version'] = $git_version . '-dev';
         }
       }
+      else {
+        $modules[$key]->info['version'] = 'unknown';
```

Comment:
@alanmels thank you so much for your patience. :smiley:

Here's yet another request: as we have a plain string here, it could be translatable. But be warned, this function is also called from the installer, so you would have to do it like that:

```
$t = get_t();
$t('unknown');
```

The docs: https://docs.backdropcms.org/api/backdrop/core%21includes%21bootstrap.inc/function/get_t/1
What do you think?

Reply:
Ah, I always forget about making strings translatable. Corrected. Thanks for noting.